src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingActualsResponseMapper.java

```diff
@@ -237,17 +233,14 @@ private OrganisationDetails setOrganisationDetails(HearingPartyEntity hearingPar
         return organisationDetails;
     }
 
-    private ArrayList<IndividualDetails> setIndividualDetails(HearingPartyEntity hearingPartyEntity) {
-        ArrayList<IndividualDetails> individualDetailsArrayList = new ArrayList<>();
+    private IndividualDetails setIndividualDetails(HearingPartyEntity hearingPartyEntity) {
+        IndividualDetails individualDetails = new IndividualDetails();
```

Comment:
```suggestion
        IndividualDetails individualDetails = null;
```
Should probably instantiate inside the `if`, so it's only when required?

Reply:
changed